Floofbot/Modules/ModMail.cs

```diff
@@ -118,15 +116,23 @@ private void CheckServerEntryExists(ulong server)
                     ModRoleId = null
                 });
                 _floofDB.SaveChanges();
+                return _floofDB.ModMails.Find(server);
```

Comment:
This isn't _quite_ what I had in mind. I was thinking of using the result of `_floofDB.Add` as the function's return value. Would that not work?